src/lando/ui/pull_requests.py

```diff
@@ -21,10 +21,12 @@ def get(
         client = GitHubAPIClient(target_repo)
         pull_request = PullRequest(client.get_pull_request(number), target_repo)
 
+        # NOTE: landing_jobs added for template compatibility.
```

Comment:
nit: We could a `|default()` in the template https://github.com/mozilla-conduit/lando/blob/557d1f185644898106f478bb7a20ab6e8f238a4c/src/lando/ui/jinja2/stack/partials/timeline.html#L2 instead, so we don't need to worry about it?

That said, it's a nice summary anyway, so I'd say it is desirable to have.